examples/wfs.js

```diff
@@ -31,6 +31,7 @@ view.addLayer({
     networkOptions: { crossOrigin: 'anonymous' },
     type: 'color',
     protocol: 'wms',
+    disableGetCap: true, // their getCap call is too slow
```

Comment:
Can't this be implicit? If the user supplies all needed paramaters (format, version?) then you assume that GetCapabilities is not needed.
WDYT?

Reply:
It's an interesting idea. The advantage of `disableGetCap` is that it *explicitly* ensure getCap won't be called. If it's implicit, it can still be called if you forget something. Maybe we can cover both use cases ? if we have all the parameters, we disable it. If we have disableGetCap, we check the needed parameters and throw if one is missing. WDYT?

Also, WDYT about `disableGetCap` naming? The more I see it, the more I prefer `disableGetCapabilities`.

Reply:
Your suggestion seems fine.

And I prefer `disableGetCapabilities` (or `noGetCapabilities`?)